CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 
 #### 5.x Releases
 
-- `5.0.x` Releases - [5.0.0](#500)
+- `5.0.x` Releases - [5.0.0](#500)||[5.0.1](#501)
```

Comment:
`[5.0.0](#500)||[5.0.1](#501)` -> `[5.0.0](#500) | [5.0.1](#501)`. It's just for markdown formatting.

Reply:
Done